tools/mobrpg/tests/test_packaging.py

```
@@ -0,0 +1,86 @@
+    any in-process check noticing, since importlib.resources only sees
+    what's on disk in this checkout, not what setuptools decides to ship."""
+    project_root = pathlib.Path(__file__).resolve().parents[1]
+    # setuptools' build_py caches copies in build/lib and doesn't prune files
+    # that were deleted from source between builds, which would leak stale
+    # _legacy/* into this wheel even after the source dir is clean. Both dirs
+    # are gitignored scratch output, safe to blow away before a fresh build.
+    for stale in ("build", "mobrpg_cli.egg-info"):
+        stale_path = project_root / stale
+        if stale_path.exists():
+            shutil.rmtree(stale_path)
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)